lib/order.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'menu'
```

Comment:
WWW: This class is written very well, EBI: You had a 'order loop' method which called these methods